pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -62,10 +62,11 @@ const (
 	ciDirname  = runDirname + "/cloudinit" // For cloud-init images
 
 	// Time limits for event loop handlers
-	errorTime           = 3 * time.Minute
-	warningTime         = 40 * time.Second
-	casClientType       = "containerd"
-	unknownStateRetries = 10
+	errorTime               = 3 * time.Minute
+	warningTime             = 40 * time.Second
+	casClientType           = "containerd"
+	unknownStateRetries     = 10
+	LocalActiveAppConfigDir = "/persist/vault/active-app-instance-config/"
```

Comment:
Yetus says this needs a comment. Also, please put it in types/locationsconsts.go to avoid duplicating it in two files.